### DISCUSSION TITLE
Multi tag lower / upper case

### DISCUSSION THREAD

**Discussion by headmidtail** [DeaDBeeF-Player/deadbeef#3017] [Q&A] [answered]
Since deadbeef partly supports FB2K title formatting, is it already possible to lower / upper case all selected tags? Example: something like properties->metadata->edit and reformat TITLE = $lower(%title%)

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
Deadbeef doesn't support title formatting in the tag editor.
TBH I didn't know foobar2000 does :)

**Reply by headmidtail** [reply]
Oh, what a pity, but maybe it will be implemented at any time. foobar2000s tagging is very powerful and afaik the only tool that can do above operation.
